content/knowledge/consensus_algorithms.md

```diff
@@ -0,0 +1,367 @@
+In the next step, each of the servers then vote on the accuracy of the transactions.
+
+Transactions with more than a certain percentage of `no` — votes are either discarded or included in the next candidate set for the next consensus round.
+If at least 80% of a UNL list agrees on a transaction, it is applied to the ledger.
+The ledger is closed and becomes the new last-closed ledger.
+
+In Ripple, the notion of _majority rule_ is applied on the UNL set requirement for agreement.
+From a high level this system is a group of overlapping groups of nodes, because some of the nodes in one UNL list are present in another UNL for another node.
+The notion of hard majority cannot be applied in Ripple, because the total number of servers in the system is unknown, consequently there is no hard number of voters for any of the participants.
+
+Each of the servers can join or leave the network at some point, which means that in some of the consensus rounds some servers will have different nodes in their UNL lists.
+
+## Avalanche and How It Uses Sampling to Achieve Deterministic Probabilistic Consensus on a Leaderless Manner that Can Scale
+
+### Avalanche
+
+After Nakamoto consensus algorithm, the world still wanted to have an algorithm that does have its benefits like full decentralization, but also speed and energy efficiency.
+To solve this, Avalanche was proposed by a pseudonymous group called [Team Rocket](https://medium.com/@mr_davis_12/avalanche-and-the-team-rocket-chronology-mystery-and-pseudonymity-30c17331f6a2).
+[Avalanche](https://assets.website-files.com/5d80307810123f5ffbb34d6e/6009805681b416f34dcae012_Avalanche%20Consensus%20Whitepaper.pdf) is a probabilistic consensus protocol based on Proof Of Stake.
+Its most important property, which sets it apart from most of existing protocols, is that like Nakamoto consensus, it can operate with no known upper-bound of participants in the network.
+It aims to reach the decentralization level of Nakamoto, resilience to 51% attacks, as well as high throughput and low latency.
+It follows the idea of having some neglectable error when reaching of consensus as Nakamoto suggested, which makes it probabilistic and means there is no 100% certainty of reaching a consensus.
+
+The number of messages each node has to handle per decision is with O(k) complexity and does not grow as the network scales up, consequently the system can scale to millions of validators all participating in the consensus to achieve a global size decentralized blockchain network.
+
+Logically, Avalanche is a validator protocol. Each node has a known number of validators (other nodes) called _validator list_.
+Each of the validators stakes some amount in the network and the probability of their selection in the lists of other nodes is weighted by this staked amount.
+Validators listen for transactions.
+When they see a new transaction they vote on whether it is to be `Accepted` or `Rejected`.
```

Comment:
Why not just `whether it is to be accepted or rejected`?

Reply:
Yes, this has been corrected https://github.com/optakt/optakt.github.io/pull/21/commits/ca80e6c74bee64b8ce712ca515bb205ee645d971#diff-4bf8323c6426a93da1ce22be1b8528779ef6b61fdacef3e0d065f1e1dd47bea5R22
Thanks!